source/user-manual/api/reference.rst

```diff
@@ -7194,7 +7544,7 @@ Returns the sca database of an agent.
 **Example Request:**
 ::
 
-	curl -u foo:bar -k -X GET "https://127.0.0.1:55000/sca/000?q=pass>50;score<150&pretty"
+	curl -u foo:bar -k -X GET "https://127.0.0.1:55000/sca/000?q=pass>;score<150&pretty"
```

Comment:
I don't understand this example. Why has been removed the '50'?

Reply:
Fixed!